### PR TITLE
[rllib] add tuned example for pendulum

### PR DESCRIPTION
**Pull request by ericl**
https://github.com/ray-project/ray/issues/1551

### CODE REVIEW

File: python/ray/rllib/ppo/ppo.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
     "timesteps_per_batch": 4000,
     # Each tasks performs rollouts until at least this
     # number of steps is obtained
-    "min_steps_per_task": 1000,
+    "min_steps_per_task": 200,
```
Comment (ericl) [inline python/ray/rllib/ppo/ppo.py]: 1000 seems to trip up too many experiments... 200 is a safer value and shouldn't be much of a hit

File: python/ray/rllib/tuned_examples/pendulum-ppo.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# can expect improvement to -140 reward in ~300-500k timesteps
+hopper-ppo:
+    env: Pendulum-v0
+    run: PPO
+    resources:
+        cpu: 4
+    config: {"timesteps_per_batch": 2048, "lambda": 0.1, "gamma": 0.95, "sgd_stepsize": 0.0003, "sgd_batchsize": 64, "num_sgd_iter": 10, "model": {"fcnet_hiddens": [64, 64]}, "min_steps_per_task": 100}
```
Comment (richardliaw) [inline python/ray/rllib/tuned_examples/pendulum-ppo.yaml]: Can you lay this out in yaml? It's hard to read this

File: python/ray/rllib/tuned_examples/pendulum-ppo.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# can expect improvement to -140 reward in ~300-500k timesteps
+hopper-ppo:
```
Comment (richardliaw) [inline python/ray/rllib/tuned_examples/pendulum-ppo.yaml]: `pendulum-ppo`

**Review by richardliaw — CHANGES_REQUESTED** [review]

File: python/ray/rllib/tuned_examples/pendulum-ppo.yaml (hunk 2 above)
Comment (ericl) [reply python/ray/rllib/tuned_examples/pendulum-ppo.yaml]: Done

File: python/ray/rllib/tuned_examples/pendulum-ppo.yaml (hunk 3 above)
Comment (ericl) [reply python/ray/rllib/tuned_examples/pendulum-ppo.yaml]: ray.init(redis_address="192.168.5.113:50598")

**Review by richardliaw — APPROVED** [review]

**Comment by richardliaw** [comment]
failures not related

**Pull request merged**